Flank/src/main/java/com/walmart/otto/reporter/TimeReporter.java

```diff
@@ -28,4 +28,8 @@ public static String getEndTime() {
         Date date = new Date();
         return simpleDateFormat.format(date);
     }
+
+    public static List<Integer> getExecutionTimes(){
+        return executionTimes;
+    }
 }
```

Comment:
I have an open issue: https://github.com/TestArmada/flank/issues/24

java has a robust linting ecosystem and the better ones auto correct issues automatically. I will probably suggest adopting the Google Java format:

- https://github.com/google/google-java-format

this is a single command that updates all the code. 😄